backend/contracts/IdentityAggregator.sol

```diff
@@ -0,0 +1,119 @@
+    event RemoveSupportForContract(string contractKeyword);
+
+
+    constructor() {
+        
+    }
+
+
+    /// @notice Add support for a new VerifyJWT contract.
+    /// @param keyword The string used to denote the source of the JWT (e.g., "twitter").
+    /// @param contractAddress The address of the JWT contract to be supported.
+    function addPlatformContract(string memory keyword, address contractAddress) public onlyOwner { // TODO: there must be a better way than onlyOwner
+
+        // Require that neither this keyword nor this contract has been added
+        require(bytes(keywordForKeyword[keyword]).length == 0);
+        require(contractAddrForKeyword[keyword] == address(0));
+
+        keywords.push(keyword);
+        keywordForKeyword[keyword] = keyword;
+        contractAddrForKeyword[keyword] = contractAddress;
+
+        emit AddSupportForContract(keyword);
+    }
+
+    // TODO: Is there a way to store keywords that allows iteration but allows removal without iteration?
+    /// @notice Remove support for a VerifyJWT contract.
+    /// @param keyword The string used to lookup the contract.
+    function removeSupportFor(string memory keyword) public onlyOwner { // TODO: there must be a better way than onlyOwner
+        require(contractAddrForKeyword[keyword] != address(0), "There is no corresponding contract for this keyword.");
+        
+        for (uint i = 0; i < keywords.length; i++) {
+            if (keccak256(bytes(keywords[i])) == keccak256(bytes(keyword))) {
+                contractAddrForKeyword[keyword] = address(0);
+                keywordForKeyword[keyword] = "";
+                keywords[i] = "";
+
+                emit RemoveSupportForContract(keyword);
+                break;
+            }
+        }
+    }
+
+    /// @notice For user, get creds for every platform designated in keywords.
+    /// @param user The address whose creds will be returned.
+    /// @return creds A list of creds corresponding to user.
+    function getAllAccounts(address user) public returns (bytes[] memory creds) {
+        bytes[] memory allCreds = new bytes[](keywords.length);
+        for (uint i = 0; i < keywords.length; i++) {
+            if (bytes(keywords[i]).length != 0) {
+                address contractAddr = contractAddrForKeyword[keywords[i]];
+                VerifyJWT contract_ = VerifyJWT(contractAddr);
+                bytes memory credsTemp = contract_.credsForAddress(user);
+                if (credsTemp.length != 0) {
+                    allCreds[i] = credsTemp;
+                }
+            }
+        }
+        return allCreds;
+    }
+
+    // TODO: Either delete this function, or find a better implementation.
+    // e.g., keyword1 == "orcid", creds1 == "12345...", keyword2 == "twitter" ---> returns "@somehandle"
+    function getCredsFromCreds(
+        string memory keyword1, 
+        bytes calldata creds1, // TODO: Make these parameters fixed-length
```

Comment:
I actually don't think it's feasible to make them fixed length, since VerifyJWT needs creds to be of type bytes and has no built-in enforcement of length of its bytes. However, we could change that contract too if necessary so it also enforces a byte limit